sass.py

```diff
@@ -694,12 +694,15 @@ def __new__(cls, r, g, b, a):
 SEPARATORS = frozenset((SASS_SEPARATOR_COMMA, SASS_SEPARATOR_SPACE))
 
 
-class SassList(collections.namedtuple('SassList', ('items', 'separator'))):
+class SassList(collections.namedtuple(
+        'SassList', ('items', 'separator', 'bracketed'),
+)):
 
-    def __new__(cls, items, separator):
+    def __new__(cls, items, separator, bracketed):
```

Comment:
yeah I considered that as well but wasn't really sure what to put.  Both values seem equally wrong to me (though `False` was the *only* possible value before today).

On the other hand, it's super unlikely that consumers are even using this feature.  At least internally we're only using sass functions to return strings and don't come close to dealing with maps or lists.

I guess I could change this to `bracketed=False`?